dask/dataframe/core.py

```diff
@@ -1751,7 +1754,10 @@ class Series(_Frame):
 
     def __array_wrap__(self, array, context=None):
         if isinstance(context, tuple) and len(context) > 0:
-            index = context[1][0].index
+            if isinstance(context[1][0], np.ndarray) and context[1][0].shape == ():
```

Comment:
This is getting *close* to enough code where we might want to refactor it to a base method. In pandas this would be a `_shallow_copy`. You pass it the data, and the result is constructed with all the metadata and indexes.

For now though, it's probably fine.